cwf/gateway/configs/pipelined.yml

```diff
@@ -157,7 +157,7 @@ has_quota_port: 51115
 no_quota_port: 51125
 
 # For 5G Functionality Support flag
-5G_feature_set:
+enable_5G_feature_set:
```

Comment:
why do we need the word `enable` here if we already have a flag called enable?

Isn't it better to leave it as 5g feature set?

Reply:
in case you can't use a number at the very begging, just put 5g at the end
```feature_set_5g```